### DISCUSSION TITLE
Unable to update Spec pattern for one of the projects

### DISCUSSION THREAD

**Discussion by brahmini** [cypress-io/cypress#22553] [Questions and Help]
Hello, I'm looking for some help here

**Question**: Does Cypress V10 support executing multiple projects?

I have 2 different cypress projects and currently have trouble migrating spec patterns for one of them. The first project had no issues after the v10 update (I'm able to access spec files and execute tests) but the second one says:

```We couldn't find any files matching the following spec patterns. You may need to move your specs or update your specPattern to find them.```

<img width="1024" alt="Screen Shot 2022-06-28 at 1 49 08 AM" src="https://user-images.githubusercontent.com/16943895/176141405-cceda2f5-3f5f-4ae8-9fb0-7e52bcfcf7cb.png">

--> I tried clicking the `Update Spec Pattern` option and it pulled the project but nothing happened.

**Note**: I included `specPattern: 'cypress/e2e/**/*.{js,jsx,ts,tsx}'` within `cypress.config.js` manually as it wasn't included during migration earlier.

**Additional context related to projects**:
1. One project includes website E2E testing and
2. The other project includes automated analytics testing, so the file format looks as below:
    * sampleTest.js
    * sampleTest.micro.js

Your help is appreciated, thank you in advance!

**Comment by portyu9** [comment]
Hi @brahmini 

Cypress v10+ supports multiple projects, but each project should have its own `cypress.config.js` and be opened/run from the correct project root.

For the second project, explicitly configure the directory containing those analytics specs:

```js
const { defineConfig } = require('cypress')

module.exports = defineConfig({
  e2e: {
    specPattern: [
      'cypress/e2e/**/*.js',
      'path/to/analytics/**/*.js',
    ],
  },
})
```

Then open that project explicitly:

```bash
npx cypress open --project path/to/second-project
```

`sampleTest.micro.js` already matches `**/*.js`, so the `.micro.js` suffix is not the problem.

The fix is to make `specPattern` match the **actual location of the files** and launch Cypress with the correct project root.

Please mark this answer as accepted if it helped.
